app/assets/javascripts/discourse/app/components/modal/login/footer.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="modal-footer">
+  {{#if @canLoginLocal}}
+    {{#unless @showSecurityKey}}
+      <DButton
+        @action={{@login}}
+        id="login-button"
+        @form="login-form"
+        @icon="unlock"
+        @label={{@loginButtonLabel}}
+        @disabled={{@loginDisabled}}
+        class="btn btn-large btn-primary"
+        @tabindex="2"
+      />
+    {{/unless}}
+
+    {{#if @showSignupLink}}
+      <DButton
+        class="btn-large"
+        id="new-account-link"
+        @action={{@createAccount}}
+        @label="create_account.title"
+        @tabindex="3"
+      />
+    {{/if}}
+  {{/if}}
+  <ConditionalLoadingSpinner @condition={{@loggingIn}} @size="small" />
```

Comment:
Is there a UI change with the spinner? Not against it, but curious, because previosuly the login button itself would switch its icon to a spinner (and it would switch the label as well).

Reply:
I don't believe there is any UI change here

##### Before
<img width="291" alt="Screenshot 2023-08-17 at 12 50 29 PM" src="https://github.com/discourse/discourse/assets/50783505/535c9db9-ecc8-4908-a64b-be5d8ba399e9">

##### After
<img width="281" alt="Screenshot 2023-08-17 at 12 51 03 PM" src="https://github.com/discourse/discourse/assets/50783505/50426fbd-20b4-49fe-8794-2432fcd0949b">